### PR TITLE
PRO-2734: Prevent example dates being rendered as links on cellular devices

### PR DESCRIPTION
**Pull request by adamb-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-2734

### Change description ###
Prevent example dates being rendered as links on cellular devices.
This change adds the helper CSS class of .no-link which prevents any link from being and looking clickable.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app/steps/ui/will/date/template.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1 class="form-title heading-large">{{content.question}}</h1>
     <div class="panel panel-border-narrow js-hidden" id="isWillDate-1-details">
         <h2 class="form-title heading-medium">{{content.willDateQuestion}}</h2>
         <p>{{content.willDateExplanation}}</p>
-        <p class="form-hint">{{content.willDateHint}}</p>
+        <p class="form-hint no-link">{{content.willDateHint}}</p>
```
Comment (shepda) [inline app/steps/ui/will/date/template.html]: Can we add the form hint into the `date` mixin?

File: app/steps/ui/will/codicilsdate/template.html [hunk 2]
```diff
@@ -28,7 +28,7 @@ <h1 class="form-title heading-large">{{content.question | safe}}</h1>
         <div class="panel panel-border-narrow js-hidden" id="yes-details">
             <h2 class="form-title heading-medium">{{content.codicilsDateQuestion}}</h2>
             <p>{{content.codicilsDateExplanation}}</p>
-            <p class="form-hint">{{content.codicilsDateHint}}</p>
+            <p class="form-hint no-link">{{content.codicilsDateHint}}</p>
```
Comment (shepda) [inline app/steps/ui/will/codicilsdate/template.html]: Can we add the form hint into the `date` mixin?

File: app/steps/ui/deceased/dod/template.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 {% block question %}
     <h1 class="form-title heading-large">{{content.question}}</h1>
-    <p class="form-hint">{{content.questionHint}}</p>
+    <p class="form-hint no-link">{{content.questionHint}}</p>
```
Comment (shepda) [inline app/steps/ui/deceased/dod/template.html]: Can we add the form hint into the `date` mixin?

File: app/steps/ui/deceased/dob/template.html [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 {% block question %}
     <h1 class="form-title heading-large">{{content.question}}</h1>
-    <p class="form-hint">{{content.questionHint}}</p>
+    <p class="form-hint no-link">{{content.questionHint}}</p>
```
Comment (shepda) [inline app/steps/ui/deceased/dob/template.html]: Can we add the form hint into the `date` mixin?

Comment (adamb-moj) [reply app/steps/ui/deceased/dob/template.html]: Hi @shepda 
They've been moved into the date mixin now.

**Pull request merged**